manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -133,7 +133,7 @@ func (r *FybrikApplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	observedStatus := application.Status.DeepCopy()
 	appVersion := application.GetGeneration()
 
-	// validate fybrik application in case of the create/update resource event
+	// validate fybrik application in case of the creation/update resource event
```

Comment:
// validate fybrik application if the resource has been created or modified

Reply:
OK